ui/components/multichain/asset-picker-amount/hooks/useIsFiatPrimary.tsx

```diff
@@ -0,0 +1,17 @@
+import { useSelector } from 'react-redux';
+import {
+  getPreferences,
+  getSendInputCurrencySwitched,
+} from '../../../../selectors';
+
+export default function useIsFiatPrimary() {
+  const { useNativeCurrencyAsPrimaryCurrency } = useSelector(getPreferences);
```

Comment:
Thoughts on writing this as a memoized selector instead of a custom hook? The logic for calculating `isFiatPrimary` seems simple enough to derive directly from redux state

Reply:
Great call!